imsv-docs-docusaurus/openapi/endpoints/login/models/create-challenge-response.yaml

```diff
@@ -0,0 +1,39 @@
+type: object
+required:
+  - purpose
+  - address
+  - network
+properties:
+  id:
+    description: The unique identifier of the challenge.
+    type: string
+    example: "GlSDzZHcxrSDeJbKu"
+  purpose:
```

Comment:
we should probably also have an enum of supported purposes

Reply:
Done 👍